presto-main/src/test/java/com/facebook/presto/operator/TestHashAggregationOperator.java

```diff
@@ -657,6 +677,64 @@ public void testSpillerFailure()
         }
     }
 
+    @Test
+    public void testMemoryPools()
```

Comment:
ditto regarding renaming this to `testSystemMemoryUsage`

Reply:
Ditto, `testMemoryTracking`